File: program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java
public class PrintNumbersFrom1ToN{
    public static void main(String[] args) {
       Scanner sc = new Scanner(System.in);
       System.out.print("Enter the number: ");
       int n = sc.nextInt();
       print(n);
  }
  static void print(int n){
    for(int i = 1; i <=n; i++){
      System.out.print(i + " ");
    }
  }
}

File: program/print-numbers-from-1-to-n/printNumbersFrom1ToN.scala
import scala.io.StdIn._;

object print1ToN {
    def main(args:Array[String]):Unit={
        println("Enter a number : ");
        var n = readLine().toInt;
        var cnt:Int=0;
        
        cnt=1;
        while(cnt<=n)
        {
            printf("%d ",cnt);
            cnt=cnt+1;
        }
    }
}
